proto/viam/app/v1/app.proto

```diff
@@ -1411,3 +1423,77 @@ message CreateKeyFromExistingKeyAuthorizationsResponse {
   string id = 1;
   string key = 2;
 }
+
+message OrganizationSetLogoRequest {
+  string org_id = 1;
+  bytes logo = 2;
+}
+message OrganizationSetLogoResponse {}
+
+message OrganizationGetLogoRequest {
+  string org_id = 1;
+}
+message OrganizationGetLogoResponse {
+  string url = 1;
+}
+
+message EnableAuthServiceRequest {
+  string org_id = 1;
+}
+message EnableAuthServiceResponse {}
+
+message DisableAuthServiceRequest {
+  string org_id = 1;
+}
+message DisableAuthServiceResponse {}
+
+message CreateOAuthAppRequest {
+  string org_id = 1;
+  string name = 2;
+  OAuthConfig oauth_config = 3;
+}
+message CreateOAuthAppResponse {
+  string id = 1;
+  string secret = 2;
+}
+
+message ReadOAuthAppRequest {
+  string org_id = 1;
+  string id = 2;
+}
+message ReadOAuthAppResponse {
+  string name = 1;
+  string secret = 2;
+  OAuthConfig oauth_config = 3;
+}
+
+message UpdateOAuthAppRequest {
+  string org_id = 1;
+  string id = 2;
+  string name = 3;
+  OAuthConfig oauth_config = 4;
+}
+message UpdateOAuthAppResponse {}
+
+message DeleteOAuthAppRequest {
+  string org_id = 1;
+  string id = 2;
+}
+message DeleteOAuthAppResponse {}
+
+message ListOAuthAppsRequest {
+  string org_id = 1;
+}
+message ListOAuthAppsResponse {
+  repeated string ids = 1;
+}
+
+message OAuthConfig {
+  string client_authentication = 1;
+  string pkce = 2;
+  string url_validation = 3;
```

Comment:
what are example values of these? for my own knowledge

Reply:
`client_authentication` can be one of "Required", "NotRequired", or "NotRequiredWhenUsingPKCE"
`pkce` can be one of "Required", "NotRequired", or "NotRequiredWhenUsingClientAuthentication"
`url_validation` can be one of "ExactMatch" or "AllowWildcards"

(... those strings are exactly how its defined in the FusionAuth API, but maybe we can expose it to the user a little differently and match to the FusionAuth values we will need ... )

Reply:
should we make enums for that?

Reply:
Yes we can good idea!